diff --git a/LICENSE b/LICENSE
old mode 100644
new mode 100755
diff --git a/README.md b/README.md
old mode 100644
new mode 100755
diff --git a/index.js b/index.js
old mode 100644
new mode 100755
diff --git a/package.json b/package.json
old mode 100644
new mode 100755
diff --git a/simple-markdown.d.ts b/simple-markdown.d.ts
old mode 100644
new mode 100755
diff --git a/simple-markdown.js b/simple-markdown.js
old mode 100644
new mode 100755
index d2dbac2b52c502919ee1eb587452dc3bea61a09f..02ae5b3794adbd8a19d6cc7ad55b87d3d52aa6f7
--- a/simple-markdown.js
+++ b/simple-markdown.js
@@ -526,7 +526,7 @@ var anyScopeRegex = function(regex /* : RegExp */) {
 
 var TYPE_SYMBOL =
     (typeof Symbol === 'function' && Symbol.for &&
-     Symbol.for('react.element')) ||
+     Symbol.for('react.transitional.element')) ||
     0xeac7;
 
 /**
@@ -546,7 +546,8 @@ var reactElement = function(
         key: key == null ? undefined : key,
         ref: null,
         props: props,
-        _owner: null
+        _owner: null,
+        _store: { validated: 0 }
     } /* : any */);
     return element;
 };
diff --git a/simple-markdown.min.js b/simple-markdown.min.js
old mode 100644
new mode 100755
